src/acquisition/covidcast/database.py

```diff
@@ -319,8 +321,9 @@ def worker():
               dict(zip(w_cursor.column_names, x)) for x in w_cursor
             ))
           srcsigs.task_done()
+          logger.info("completed pair", thread=name, pair=f"({source}, {signal})")
```

Comment:
this might be good to print before the sql is evaluated?

Reply:
as in a "starting pair" instead?